cmd/kubelet/app/options/options.go

```diff
@@ -115,6 +116,9 @@ type KubeletFlags struct {
 	// is true and upon the initial registration of the node.
 	RegisterWithTaints []core.Taint
 
+	// cAdvisorPort is the port of the localhost cAdvisor endpoint (set to 0 to disable)
+	CAdvisorPort int32
```

Comment:
Pros/cons regarding using a pointer or not here in the internal code?
Are we using the int32 to be consistent with most other fields or would uint16 make sense for this field internally for now?

Reply:
Not sure of the details of how Uint16Var flags get parsed. Do you get an error if you overflow it? I'd imagine they'd just use strconv.ParseUint under the hood, which should just give you a "value out of range" error? If so we could skip this in our validation.